maps/map_files/FOP_v3_Sciannex/Fiorina_SciAnnex.dmm

```diff
@@ -13300,7 +13300,8 @@
 /obj/item/clothing/under/marine{
 	desc = "A Unified Clown Federation combat uniform. Features the latest in anti-mime technology.";
 	icon_state = "clown";
-	name = "\improper UCF uniform"
+	name = "\improper UCF uniform";
+	flags_atom = 3072
```

Comment:
`NO_NAME_OVERRIDE|NO_SNOW_TYPE` ? 
I don't think having raw bitfield values in the map files is going to be very futureproof. As we saw recently with ammunition, flags are bound to change and this will be completely forgotten.

What's the intent here, is it because of the snow variations? Could make a subtype instead

Reply:
Ah, sounds good. Turned it into subtype instead.